projects/plugins/jetpack/extensions/blocks/button/test/controls.js

```diff
@@ -1,7 +1,7 @@
 /**
```

Comment:
This file had a mix of tab and space indentation - not sure why that wasn't picked up by linting when it was first added. The non whitespace related change is 154-157